packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -1427,7 +1372,7 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
    * Handles input from the key up events on the keyboard.
    */
   private onUserKeyUp(event: KeyboardEvent): void {
-    const isPaste = (event.ctrlKey || event.metaKey) && event.key === 'v';
+    const isPaste = (event.ctrlKey || event.metaKey) && event.code === 'KeyV';
```

Comment:
This change made me go learn stuff, so that's cool.
https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent#instance_properties

The warning here on the code property makes me thing that we should prefer to use .key as we don't actually know what keyboard layout a user has, QWERTY is the most common, but we should be a bit more defensive IMO

Reply:
I learnt a thing too. I'm updating to .key for this component and then for the rest of the code base I'll create an issue to refactor.